Fix usage of steam, added logging of json errors

kammerjaeger opened the pull request:
Hi,
Please have a look at the change, currently the wrong stream is used so no errors are shown.
I also added a json error output.

Let me know what you think.
It would also be nice to have a command line option to forward the input to stdout without a script or code around it.

mcollina commented on cli.js:
```diff
@@ -34,7 +34,10 @@ function start (opts) {
   const stream = pinoElasticSearch(opts)
-
+    
+  stream.on('unknown', async (error) => {
```
```suggestion
  stream.on('unknown', (error) => {
```

Why was this async?

kammerjaeger replied:
Sorry that was my mistake, to much async programming :-)

mcollina reviewed (commented):
in what circumstances `unknown` is emitted?

kammerjaeger commented:
According to documentation (and looking into the source code):
"Event received by pino-elasticsearch is unparseable (via JSON.parse)"
I fixed and added the line in the error message.

mcollina reviewed (approved):
lgtm

kammerjaeger commented:
Looks like the ';' is not liked in the linter, I will remove them.
